src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -138,27 +138,27 @@ private static String getNonNullEnvironmentVariable(String envVarName) {
 
     /**
      * @return an arguments List containing the apiKey, project, and staging arguments
-     *         populated from environment variables as defined in {@link #getDataflowTestApiKey},
-     *         {@link #getDataflowTestProject}, and {@link #getDataflowTestStaging}, suitable
+     *         populated from environment variables as defined in {@link #getGCPTestApiKey},
+     *         {@link #getGCPTestProject}, and {@link #getGCPTestStaging}, suitable
      *         for use in a hellbender command line.
      */
-    public static List<String> getStandardDataflowArgumentsFromEnvironment() {
-        return Arrays.asList("--apiKey", getDataflowTestApiKey(),
-                             "--project", getDataflowTestProject(),
-                             "--staging", getDataflowTestStaging());
+    public static List<String> getStandardGCPArgumentsFromEnvironment() {
```

Comment:
This method is only used in 1 place in `ReadsPreprocessingPipelineIntegrationTest`  Since it's really dataflow specific I think we should inline it there.